stock-filters/ChirpVillage/Builder.py

```diff
@@ -0,0 +1,66 @@
+import utilityFunctions
+import random
+
+
+class Building(object):
+
+    def construct(self, level, box):
+        # This is assumes box is already the size of the house
+        pillar_height = box.miny + random.randrange(5, 10)
+        for y in range(box.miny, pillar_height):
+            for x in [box.minx, box.maxx-1]:
+                for z in [box.minz, box.maxz-1]:
+                    utilityFunctions.setBlock(level, (1, 0), x, y, z)
```

Comment:
This should use blocks['Stone'] in block utils or using get_floor etc. I dont think we should be hardcoding blocks